styles/styles.css

```diff
@@ -0,0 +1,41 @@
+* {
+  margin: 0;
+  /*box-sizing: border-box;*/
+}
+
+.container {
+  width: 1440px;
+  margin: 0 auto;
+  padding: 120px 567px 20px;
```

Comment:
Rather huge left/right paddings. Did you saw it in the design?

Reply:
I can't connect the fonts, but I did everything from the lesson 5.
Can you check the reason?

Reply:
Yes, changed

I can't connect the fonts, but I did everything from the lesson 5.
Can you check the reason?

Reply:
You had applied it to the non-existent element.
And please check your work with perfect pixel before I view it next time.